src/ui/timers_main_area.ui

```diff
@@ -619,32 +650,9 @@
     </widget>
    </item>
    <item row="2" column="0">
-    <widget class="QLabel" name="label_4">
-     <property name="sizePolicy">
-      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
-       <horstretch>0</horstretch>
-       <verstretch>0</verstretch>
-      </sizepolicy>
-     </property>
-     <property name="minimumSize">
-      <size>
-       <width>0</width>
-       <height>0</height>
-      </size>
-     </property>
-     <property name="maximumSize">
-      <size>
-       <width>16777215</width>
-       <height>16777215</height>
-      </size>
-     </property>
-     <property name="font">
-      <font>
-       <pointsize>9</pointsize>
-      </font>
-     </property>
+    <widget class="QLabel" name="label">
      <property name="text">
-      <string>Command:</string>
+      <string>Time:</string>
```

Comment:
The example you quoting seems to suggest changing `Time:` to `Command:` I suspect that is actually a side effect of item reordering within the `.ui` file.  However, if you now compare all of the different item type dialogue they (should) all possess the __same__ text for the _same_ element which behaves the _same_ on different item types - the prior inconsistency is just that, __inconsistent__, IMHO.